Frontend/Pages/UserManagement/ForgotPassword.razor

```diff
@@ -35,16 +43,25 @@
 
 @code {
 
+    private bool _hasBeenProcessed;
+    private bool _sending;
+
+    [Inject]
+    HttpClient HttpClient { get; set; } = default!;
```

Comment:
`null` is probably preferable here, doesn't really matter

Reply:
actually isn't. somewhere in the msdocs I read that since the injected service shouldn't ever be null, default is preferred (or some reasoning along that line)

Reply:
`default` evaluates to `null` for reference types lol

Reply:
dont ask me. maybe I can find it again

Reply:
Is this what you meant? https://docs.microsoft.com/en-us/aspnet/core/blazor/fundamentals/dependency-injection?view=aspnetcore-6.0#request-a-service-in-a-component
If so, what that is saying is that you should not declare `HttpClient` as nullable (`HttpClient?`).
I really wonder why they'd suggest using default.. In this case there's virtually no reason that I can see to prefer it over `null`

Reply:
Yes. No clue, didn't spend time thinking about it but just accepted it :D